packages/components/src/components/form-submit-error-message/form-submit-error-message.jsx

```diff
@@ -6,7 +6,7 @@ import Text from '../text';
 
 const FormSubmitErrorMessage = ({ className, message }) => (
     <div className={classNames('dc-form-submit-error-message', className)}>
-        <Icon icon='IcAlertDanger' data_testid='form-submit-error' />
+        <Icon icon='IcAlertDanger' data_testid='form_submit_error' />
```

Comment:
using this testid in my test, that's why renamed it to use snake_case as per docs